agent.go

```diff
@@ -1304,6 +1308,19 @@ func init() {
 	}
 }
 
+func loadKernelModules() {
```

Comment:
I agree with @bergwolf - this function should return an `error` (which will help us unit test this... :)